Create chat view

kupomaru8 opened the pull request.

kupomaru8 committed:
Create chat view

hzak2xx commented on app/views/messages/_content-left.haml:
```diff
@@ -0,0 +1,9 @@
+    %i.fas.fa-edit
+    %i.fas.fa-cog
+
+  .bottom
```
グループ一覧なので `groups` がいいですね

hzak2xx commented on app/views/messages/_content-left.haml:
```diff
@@ -0,0 +1,9 @@
+
+  .bottom
+    asaki
+    %p メッセージはありません
```
タグ、クラス当てなくていいですか？

hzak2xx commented on app/views/messages/_chat.html.haml:
```diff
@@ -0,0 +1,7 @@
+.chat
+  %p.chat__user aa
+  %p.chat__date 2018/10/08
+  %p.chat__content こんにちは
```
```
  %p.chat__user aa
  %p.chat__date 2018/10/08
  %p.chat__content こんにちは
```
これでひとかたまりなのでこの親要素が欲しいです

hzak2xx commented on app/assets/stylesheets/application.scss:
```diff
@@ -0,0 +1,82 @@
+
+@import "./reset";
+
+html {
```
ここはファイル読み込みだけしましょう

kupomaru8 committed:
Update chat view

Update _content-left.haml
Update _chat.haml
Update application.scss
Create _left-content.scss
Create _right-content.scss

hzak2xx commented on app/assets/stylesheets/application.scss:
```diff
@@ -6,10 +6,16 @@
  *= require_tree .
  *= require_self
  */
+
```
これより↑消してimportしましょう。

kupomaru8 committed:
Update chat view
